README.md

```diff
@@ -12,32 +12,110 @@ On a _Windows_ machine, **Docker Desktop** comes with the necessary tools. Pleas
+
+Multiple sandbox configurations can be started in parallel. Sandbox maintains a notion of the "active" configuration, which is the most recent configuration "start"ed, or the configuration set by the "set-active" command.  If a configuration is not specified, the default action is to use the active configuration.  If no configuration is active, the default configuration will be used (currently set to 'release').
+
+In order to bring up, say, both betanet and mainnet on the same machine, users will have to provide distinct port numbers and COMPOSE_PROJECT_NAME in the two configuration files.  Once they are both started, users can point client code to the relevant ports specified to hit either betanet or mainnet. 
+
+
 ```plain
 sandbox commands:
-  up    [config]  -> start the sandbox environment.
-  down            -> tear down the sandbox environment.
-  reset           -> reset the containers to their initial state.
-  clean           -> stops and deletes containers and data directory.
-  test            -> runs some tests to demonstrate usage.
-  enter [algod||indexer||indexer-db]
-                  -> enter the sandbox container.
-  version         -> print binary versions.
-  copyTo <file>   -> copy <file> into the algod container. Useful for offline transactions & LogicSigs plus TEAL work.
-  copyFrom <file> -> copy <file> from the algod container. Useful for offline transactions & LogicSigs plus TEAL work.
-
-algorand commands:
-  logs            -> stream algorand logs with the carpenter utility.
-  status          -> get node status.
-  goal (args)     -> run goal command like 'goal node status'.
-  tealdbg (args)  -> run tealdbg command to debug program execution.
-
-special flags for 'up' command:
-  -v|--verbose           -> display verbose output when starting standbox.
-  -s|--skip-fast-catchup -> skip catchup when connecting to real network.
-  -i|--interactive       -> start docker-compose in interactive mode.
+  <up|start|resume> [config] [-v|--verbose] [-s|--skip_fast_catchup] 
+                             [-i|--interactive]
+      Start the sandbox docker project defined in the config, and set it as the 
+      active configuration. If blank, it will starts the active configuration, 
+      or failing that, default to the release configuration.  -v verbose output
+      when starting the docker project.  -s flag skips fast catchup if the user
+      is starting a non-private network and wish to skip fast cathcup.  -i flag 
+      start the docker project in interactive mode.
+
+  <down|pause|stop> [config] 
+      Stop the sandbox docker project defined in the config. If blank, it will
+      stop the active configuration, or failing that, default to the release 
+      configuration.
+
+  <reset|restart> [config] 
+      Stop and then start the sandbox docker project defined in the config
```

Comment:
This will wipe the ledger back to 0, very useful for testing to get it back in a clean state. I think its worth mentioning here.